examples/stateful_example.yaml

```
@@ -0,0 +1,115 @@
+  - input:
+      type: file
+      path: "./examples/data/words.txt"
+      format: text
+    pipeline:
+      thread_num: 2
+      processors:
+        - type: python
+          script: |
+            import json
+            import re
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)